back-end/database/migrations/2022_06_02_072248_create_choices_table.php

```diff
@@ -19,7 +19,7 @@ public function up()
                 ->constrained()
                 ->onUpdate('cascade')
                 ->onDelete('cascade');
-            $table->string('choices');
+            $table->string('choice');
```

Comment:
If you want to modify the existing database/table from previous PR. Pls create new migration. Don't modify the already existed migration.
We are doing this because during development, our application is already live. The previous changes were already uploaded to dev/staging. We might destroy the current data. So in order not to affect old data, we must create new migration file.

This is very important, if you need more explanation. Just message me.

Reply:
This is noted Sir Chev, so I will delete the old migration and create a complete migration file for the changes?

Reply:
@pong-sun 
no, dont delete.Just create a new migration file that will update only a certain column